addons/acodec/acodec.c

```diff
@@ -38,6 +41,47 @@ bool al_init_acodec_addon(void)
 #endif
 
 #ifdef ALLEGRO_CFG_ACODEC_MODAUDIO
+#if (DUMB_MAJOR_VERSION) >= 2
+   /*
+    * DUMB 2.0 offers a single loader for at least 13 formats, see their
+    * readme. The .nst format is not listed there, but Amiga NoiseTracker
+    * is supported nonetheless. It merely has no common extensions, see:
+    * https://github.com/kode54/dumb/issues/53
+    */
+   ret &= al_register_audio_stream_loader(".669", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".669", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".amf", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".amf", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".asy", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".asy", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".it", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".it", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".mod", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".mod", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".mtm", _al_load_dumb_audio_stream);
+   ret &= al_register_audio_stream_loader_f(".mtm", _al_load_dumb_audio_stream_f);
+   ret &= al_register_audio_stream_loader(".nst", _al_load_dumb_audio_stream);
```

Comment:
Didn't you just say it wasn't listed?

Reply:
I think he meant it wasn't listed in the DUMB documentation.

Reply:
Earlier upstream discussion: https://github.com/kode54/dumb/issues/53#issuecomment-330127761 -- based on that, I decided to add `.nst` in the code above.

I'm considering to remove `.nst` above because, after googling around, it seems really hard to find NST modules with that extension. Not even [modarchive](https://modarchive.org/index.php) lists NST it in their format choice.